src/Common/tests/System/PlatformDetection.cs

```diff
@@ -13,9 +13,11 @@ namespace System
     public static class PlatformDetection
     {
         public static bool IsWindows { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-        public static bool IsWindows7 { get; } =  IsWindows && GetWindowsVersion() == 6 && GetWindowsMinorVersion() == 1;
+        public static bool IsWindows7 { get; } = IsWindows && GetWindowsVersion() == 6 && GetWindowsMinorVersion() == 1;
         public static bool IsOSX { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
         public static bool IsNetBSD { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
+        public static bool IsNotWindowsNanoServer { get; } = (IsWindows && GetWindowsVersion() == 10 &&
```

Comment:
Does this property return `false` for Win7 and Win8? It means that the affected tests will be disabled for nanoserver, but also for Win7 and Win8...

I think the implementation was meant to be:
```
public static bool IsNotWindowsNanoServer { get; } = (IsWindows && (GetWindowsVersion() < 10 || File.Exists(...)));
```